SUMMARY.md

```diff
@@ -91,7 +81,7 @@
 
 ## Tutorials
 
-* [Creating a Blog](tutorials/creating-a-blog.md)
+* [Creating a Blog](creating-a-blog.md)
```

Comment:
here the tutorial was duplicated in a folder tutorials split into pages but was not used so I removed it, now we have only the creating-a-blog page with all inside (as right now) but other pages were not used